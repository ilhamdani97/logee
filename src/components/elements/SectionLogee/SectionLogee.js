import React from 'react';
import styles from './styles.scoped.css';
import PropTypes from 'prop-types';


const SectionLogee = ({ children, className }) => {
  const classes = [styles.root, className].join(' ');
  return (
    <div className={classes}>
      {children}
    </div>
  );
};


SectionLogee.defaultProps = {
  children : null,
  className: '',
};

SectionLogee.propTypes = {
  children : PropTypes.node,
  className: PropTypes.string,
};

export default SectionLogee;
