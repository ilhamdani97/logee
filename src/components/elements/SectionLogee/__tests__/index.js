import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SectionLogee from '../SectionLogee';

describe('src/components/elements/SectionLogee', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<SectionLogee />);
    expect(tree).toMatchSnapshot();
  });
});
