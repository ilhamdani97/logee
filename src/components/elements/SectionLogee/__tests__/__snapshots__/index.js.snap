// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/SectionLogee render 1`] = `
<div
  className=" "
/>
`;
